nursery/packed-with-themida.yml

```diff
@@ -7,13 +7,20 @@ rule:
     att&ck:
       - Defense Evasion::Obfuscated Files or Information::Software Packing [T1027.002]
     mbc:
-      - Anti-Static Analysis::Software Packing [F0001]
+      - Anti-Static Analysis::Software Packing::Themida [F0001.011]
     references:
       - https://www.hexacorn.com/blog/2016/12/15/pe-section-names-re-visited/
     examples:
+      - 8a132663bee5c2f0f5cbfebee1b55ac72934632bf32bc32d6e2dae797c9e6e35
+      - 2826b762b9c268601a44974ef469a671b441e798a6c3cbb40070450c6c030ba2
```

Comment:
now that we have examples, would you move this file out of the nursery?

Reply:
I moved the rule from nursery to anti-analysis/packer/themida